fri/src/two_adic_pcs.rs

```diff
@@ -389,18 +389,29 @@ where
                     .map(|&height| Dimensions { width: 0, height })
                     .collect_vec();
 
-                let batch_max_height = batch_heights.iter().max().expect("Empty batch?");
-                let log_batch_max_height = log2_strict_usize(*batch_max_height);
-                let bits_reduced = log_global_max_height - log_batch_max_height;
-                let reduced_index = index >> bits_reduced;
-
-                self.mmcs.verify_batch(
-                    batch_commit,
-                    &batch_dims,
-                    reduced_index,
-                    &batch_opening.opened_values,
-                    &batch_opening.opening_proof,
-                )?;
+                if let Some(batch_max_height) = batch_heights.iter().max() {
```

Comment:
why would we ever have an empty batch? this would be a commitment with no matrices

Reply:
resolving due to the other discussion that in principle an `Mmcs` could potentially want to handle empty batches